<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SV_RtuSendCmdDialog</class>
 <widget class="QDialog" name="SV_RtuSendCmdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>189</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>感測器名稱</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leName"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>137</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>離開</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAddr1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>97</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAddr2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>97</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>1</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLoop">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>97</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>71</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>節點</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>72</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>區</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>72</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>迴路</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>72</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>次地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>137</y>
     <width>75</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>On</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOff">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>137</y>
     <width>75</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leZone">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>97</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNode">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>97</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>97</y>
     <width>74</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地           址</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
